python-sdk/tests/databases/test_base_database.py

```diff
@@ -96,3 +98,44 @@ def test_subclass_missing_append_table_raises_exception():
     target_table = Table()
     with pytest.raises(NotImplementedError):
         db.append_table(source_table, target_table, source_to_target_columns_map={})
+
+
+def test_database_with_check_for_minio_connection():
```

Comment:
It feels like we could split this test into two and use the `patch` decorator as we've done in most python SDK tests

Reply:
Yup, done